docs/products/m3db/reference/terminology.rst

```diff
@@ -0,0 +1,19 @@
+Terminology for M3
+==================
+
+Time series
+    unique series of data identified by a set of labels, containing value(s) for specific points in time
+
+Point
+    (time, value) entry in single time series 
```

Comment:
Should we also add that it contain labels?

Reply:
Good idea